packages/api/src/index.js

```diff
@@ -25,19 +25,19 @@ import { userDIDRegister } from './routes/user-did-register.js'
 import { userTags } from './routes/user-tags.js'
 import { ucanToken } from './routes/ucan-token.js'
 import { did } from './routes/did.js'
+import { getServiceConfig } from './config.js'
 
 import {
   withMode,
   READ_ONLY as RO,
   READ_WRITE as RW,
-  DEFAULT_MODE,
   setMaintenanceModeGetter,
 } from './middleware/maintenance.js'
 import { getContext } from './utils/context.js'
 import { withAuth } from './middleware/auth.js'
 
-const getMaintenanceMode = () =>
-  typeof MAINTENANCE_MODE !== 'undefined' ? MAINTENANCE_MODE : DEFAULT_MODE
+const getMaintenanceMode = () => getServiceConfig().maintenanceMode
+
 setMaintenanceModeGetter(getMaintenanceMode)
```

Comment:
We dont need this workaround of getter and setter, this was due to a bug in `wrangler dev` that didnt load the injected globals properly. 

The maintenance middleware can access the config directly just like any other injected global.

Reply:
This is just a thing we never clean up not super related to this PR, so you can ignore it if you dont feel like changing it.

Reply:
Oh sweet, I was a bit confused by that and meant to ask you but forgot :) I'll pull it out.